apps/sms/views/conversation/js/conversation.js

```diff
@@ -723,6 +725,45 @@ var ConversationView = {
     });
   },
 
+  handleComposer: function thui_handleComposer(composer, recipients) {
+
+    function generateMmsAttachmentsForComposer(slides, content) {
+      if (typeof content === 'string') {
+        slides.push({
+          content: new Blob([content], { type: 'text/plain' }),
+        });
+      } else {
+        slides.push({
+          content: content.blob,
+          location: content.name
+        });
+      }
+      return slides;
+    }
+
+    Compose.fromMessage({
+      subject: composer.subject,
+      type: composer.type,
+      body: composer.body,
+      attachments: composer.content
+        .reduce(generateMmsAttachmentsForComposer, [])
+    });
+    Compose.focus();
+
+    if (recipients) {
+      for (var i in recipients) {
+        Utils.getContactDisplayInfo(
+          Contacts.findByAddress.bind(Contacts), recipients[i]
+        ).then(function onData(data) {
+          if (data.type !== '') {
```

Comment:
And that's the way I came up with to know if I found a contact in the user DB or not. Not very satisfying imo. Do you have better?